CefSharp.BrowserSubprocess.Core/Messaging/EvaluateScriptDelegate.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "Internals/Messaging/ProcessMessageDelegate.h"
+
+namespace CefSharp
+{
+    class CefAppUnmanagedWrapper;
+
+    namespace Internals
+    {
+        namespace Messaging
+        {
+            class EvaluateScriptDelegate : public ProcessMessageDelegate
```

Comment:
Additoinally, could new classes have nice short summary contents in their .h files explaining how the class fits into the overall picture. Like for this one: Recieves IPC requests from browser process and forwards adapts them into calls against the correct V8 JS context. (or something)